app/controllers/admin/members_controller.rb

```diff
@@ -22,6 +22,18 @@ def update
 
     private
 
+    def filter
+      @filter ||= Filter::Member.new(filter_params)
+    end
+
+    def filtered_users
+      User::Filtered.call(filter_params)
```

Comment:
I think using params/hash inside `finder` is more simple solution, so left it as it is